provisioning/transport/mqtt/src/ProvisioningTransportHandlerMqtt.cs

```diff
@@ -47,9 +49,11 @@ public class ProvisioningTransportHandlerMqtt : ProvisioningTransportHandler
         /// <summary>
         /// Creates an instance of the ProvisioningTransportHandlerMqtt class using the specified fallback type.
         /// </summary>
+        /// <param name="port">The port for the connection</param>
         /// <param name="transportFallbackType">The fallback type allowing direct or WebSocket connections.</param>
         public ProvisioningTransportHandlerMqtt(
-            TransportFallbackType transportFallbackType = TransportFallbackType.TcpWithWebSocketFallback)
+            int? port = null,
```

Comment:
Same as everywhere else - we should avoid default params. Also `port` can never be null.